android/sound.cpp

```diff
@@ -236,17 +236,10 @@ oboe::DataCallbackResult CSound::onAudioInput ( oboe::AudioStream* oboeStream, v
     // We're good to start recording now
     // Take the data from the recording device output buffer and move
     // it to the vector ready to send up to the server
-    float* floatData = static_cast<float*> ( audioData );
+    int16_t* floatData = static_cast<int16_t*> ( audioData );
```

Comment:
Is it floatData in the buffer?

Is it a pointer to integers?

Who knows?

Can you clarify this, please.

Reply:
Why is the _variable name_ `floatData`, if it's an array of `int16_t`?  That's misleading.

Reply:
Fixed.